### ISSUE TITLE
Parsing of default values may incorrectly load values that are within comments

### ISSUE THREAD

**Issue by benwmaddox** [benwmaddox/DataAccessGeneration#15]
If a comment says something like:

/*
Run dbo.ProcSample @Date = '1/2/2022'
*/
Create Proc dbo.ProcSample @Date DateTime
AS

Then it would incorrectly think @Date has a default of 1/2/2022

**Comment by benwmaddox** [comment]
Recommend waiting on #21 since it may add a better parsing approach

**Comment by benwmaddox** [comment]
No longer waiting on another issue
